DQM/DTMonitorModule/src/DTLocalTriggerLutTask.h

```diff
@@ -86,11 +87,14 @@ class DTLocalTriggerLutTask: public DQMEDAnalyzer{
   bool detailedAnalysis;
   bool overUnderIn;
 
-  edm::EDGetTokenT<L1MuDTChambPhContainer> tm_Token_;
+  edm::EDGetTokenT<L1MuDTChambPhContainer> tm_TokenIn_;
+  edm::EDGetTokenT<L1MuDTChambPhContainer> tm_TokenOut_;
   edm::EDGetTokenT<DTRecSegment4DCollection> seg_Token_;
 
-  int trigQualBest[6][5][13];
-  const L1MuDTChambPhDigi* trigBest[6][5][13];
+  int trigQualBestIn[6][5][13];
```

Comment:
Again, I'm inheriting this matrix structure from the creator... Can you be more explicit with a full example? I don't understand the structure you suggest